tests/test_models/test_losses/test_ce_loss.py

```diff
@@ -15,19 +21,56 @@ def test_ce_loss():
             loss_weight=1.0)
         build_loss(loss_cfg)
 
-    # test loss with class weights
+    # test loss with simple case for ce/bce
+    fake_pred = torch.Tensor([[100, -100]])
+    fake_label = torch.Tensor([1]).long()
     loss_cls_cfg = dict(
         type='CrossEntropyLoss',
-        use_sigmoid=False,
-        class_weight=[0.8, 0.2],
+        use_sigmoid=use_sigmoid,
         loss_weight=1.0,
+        avg_non_ignore=avg_non_ignore,
         loss_name='loss_ce')
     loss_cls = build_loss(loss_cls_cfg)
-    fake_pred = torch.Tensor([[100, -100]])
-    fake_label = torch.Tensor([1]).long()
-    assert torch.allclose(loss_cls(fake_pred, fake_label), torch.tensor(40.))
+    if use_sigmoid:
+        assert torch.allclose(
+            loss_cls(fake_pred, fake_label), torch.tensor(100.))
+    else:
+        assert torch.allclose(
+            loss_cls(fake_pred, fake_label), torch.tensor(200.))
+
+    # test loss with complicated case for ce/bce
+    # when avg_non_ignore is False, `avg_factor` would not be calculated
+    fake_pred = torch.full(size=(2, 21, 8, 8), fill_value=0.5)
+    fake_label = torch.ones(2, 8, 8).long()
+    fake_label[:, 0, 0] = 255
+    avg_factor = None
+    loss_cls = build_loss(loss_cls_cfg)
+    loss = loss_cls(fake_pred, fake_label, ignore_index=255)
+    if use_sigmoid:
```

Comment:
for bce loss, here should be to cases one is for pred.shape = label.shape, the other is for pred.shape = label.shape+1

Reply:
I think adding more here is unnecessary. L133-L134 add `fake_pred.shape = fake_label.shape+1` situation is enough.